bigquery/docs/howto/tables.rst

```diff
@@ -0,0 +1,170 @@
+Tables
+~~~~~~
+
+Tables exist within datasets. See BigQuery documentation for more information
+on `Tables <https://cloud.google.com/bigquery/docs/tables>`_.
+
+Table operations
+^^^^^^^^^^^^^^^^
+List tables for the dataset:
+
+.. literalinclude:: ../snippets.py
+   :language: python
+   :dedent: 4
+   :start-after: [START bigquery_list_tables]
+   :end-before: [END bigquery_list_tables]
+
+Create a table:
+
+.. literalinclude:: ../snippets.py
+   :language: python
+   :dedent: 4
+   :start-after: [START bigquery_create_table]
+   :end-before: [END bigquery_create_table]
+
+Get a table:
+
+.. literalinclude:: ../snippets.py
+   :language: python
+   :dedent: 4
+   :start-after: [START bigquery_get_table]
+   :end-before: [END bigquery_get_table]
+
+Update a property in a table's metadata:
+
+.. literalinclude:: ../snippets.py
+   :language: python
+   :dedent: 4
+   :start-after: [START bigquery_update_table_description]
+   :end-before: [END bigquery_update_table_description]
+
+Browse selected rows in a table:
+
+.. literalinclude:: ../snippets.py
+   :language: python
+   :dedent: 4
+   :start-after: [START bigquery_browse_table]
+   :end-before: [END bigquery_browse_table]
+
+Insert rows into a table's data:
+
+.. literalinclude:: ../snippets.py
+   :language: python
+   :dedent: 4
+   :start-after: [START bigquery_table_insert_rows]
+   :end-before: [END bigquery_table_insert_rows]
+
+Copy a table:
+
+.. literalinclude:: ../snippets.py
+   :language: python
+   :dedent: 4
+   :start-after: [START bigquery_copy_table]
+   :end-before: [END bigquery_copy_table]
+
+Extract a table to Google Cloud Storage:
+
+.. literalinclude:: ../snippets.py
+   :language: python
+   :dedent: 4
+   :start-after: [START bigquery_extract_table]
+   :end-before: [END bigquery_extract_table]
+
+Delete a table:
+
+.. literalinclude:: ../snippets.py
+   :language: python
+   :dedent: 4
+   :start-after: [START bigquery_delete_table]
+   :end-before: [END bigquery_delete_table]
+
+Upload table data from a file:
+
+.. literalinclude:: ../snippets.py
+   :language: python
+   :dedent: 4
+   :start-after: [START bigquery_load_from_file]
+   :end-before: [END bigquery_load_from_file]
+
+Load table data from Google Cloud Storage
```

Comment:
The heading logic is weird here. Why are the "Load table data from Google Cloud Storage" and "Customer Managed Encryption Keys" headers on the same level as the "Table Operations" header? I think we should either remove all of them or use headers that make more logical sense together.

Reply:
Done. Moved encryption stuff to it's own page and organized sections by operation type.